### ISSUE TITLE
sysrepoctl failed to install new yang module

### ISSUE THREAD

**Issue by dwdrajesh** [sysrepo/sysrepo#1456]
* [] Did you check if this is a duplicate issue?
It is not.
* [] Did you test it on the latest sysrepo devel branch?
Yes tested on both devel and release branch.

### Description
sysrepoctl fails to install any yang module by complaining that the corresponding /etc/sysrepo*.yang is not a directory. It does not seem to be a module-dependency issue since I tried to install simpler modules which do not import anything. The error line (in bold below) seems to be coming from the libyang/tools/lint/command.c file but does not help much.

Below is the log:
```
sudo sysrepoctl -i --yang  ./Models/ietf-interfaces.yang  -L 4

[DBG] Setting log level for stderr logs to 4.
Installing a new module from file './Models/ietf-interfaces.yang'...

Installing the YANG file to '/etc/sysrepo/yang/ietf-interfaces@2018-02-20.yang'...
**install: target '/etc/sysrepo/yang/ietf-interfaces@2018-02-20.yang' is not a directory**
Error: Unable to install the YANG file to '/etc/sysrepo/yang/ietf-interfaces@2018-02-20.yang'.
Installation of schema files cancelled for module 'ietf-interfaces', reverting...
Reverting the install operation...
Deleted the schema file '/etc/sysrepo/yang/ietf-interfaces@2018-02-20.yang'.
Install operation failed.
```
Thanks,
Rajesh

Adding @michalvasko if you have seen similar errors before? Thank you.

**Comment by michalvasko** [comment]
Hi Rajesh,
this is, indeed, quite strange and I have never seen a similar error. Could you maybe try to use an absolute path for the module? Could you provide the output of `sysrepoctl -l`?

Regards,
Michal

**Comment by dwdrajesh** [comment]
Hi Michal, 
I finally got it working . The issue was there were spaces in the directory names where the yang files and their imported modules where. For example, 
the yang file I was installing was importing another module located in a directory say "Imported module", notice the space between the "Imported" and "module". This caused libyang to fail. I renamed all those directories with an underscore instead of the space and it is working.
Does this look like a libyang issue that needs to be fixed somewhere in the library?


Thanks,
Rajesh
